src/payments/payments.module.ts

```
@@ -43,5 +43,15 @@ import { StripeProvider } from './providers/stripe/stripe.provider';
   ],
   exports: [PaymentService],
 })
-export class PaymentsModule {}
+export class PaymentsModule implements OnModuleInit {
+  private readonly logger = new Logger(PaymentsModule.name);
+
+  onModuleInit() {
+    this.logger.log('PaymentsModule initialized successfully');
```

ESLint

[error] Replace `'PaymentsModule·initialized·successfully'` with `"PaymentsModule·initialized·successfully"`

(prettier/prettier)